go/tunnels/tunnel_v.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// Generated from ../../../cs/src/Contracts/TunnelV2.cs
+
+package tunnels
+
+import (
+	"time"
+)
+
+// Data contract for tunnel objects managed through the tunnel service REST API.
+type TunnelV struct {
```

Comment:
Delete `tunnel_v.go`, `tunnel_v.rs`, `TunnelV.java`, `tunnelV.ts`. These files must have been unintentionally generated while you were working on the C# code, but would not be generated again.

Reply:
Removed